hazelcast/src/main/java/com/hazelcast/ringbuffer/ReadResultSet.java

```diff
@@ -69,4 +74,27 @@
      * @since 3.9
      */
     int size();
+
+    /**
+     * Returns the sequence of the item following the last read item. This
+     * sequence can then be used to read items following the ones returned by
+     * this result set.
+     * Usually this sequence is equal to the sequence used to retrieve this
+     * result set incremented by the {@link #readCount()}. In cases when the
+     * reader tolerates lost items, this is not the case.
+     * For instance, if the reader requests an item with a stale sequence (one
+     * which has already been overwritten), the read will jump to the oldest
+     * sequence and read from there.
+     * Similarly, if the reader requests an item in the future (e.g. because
+     * the partition was lost and the reader was unaware of this), the read
+     * method will jump back to the newest available sequence.
+     * Because of these jumps and only in the case when the reader is loss
+     * tolerant, the next sequence must be retrieved using this method.
+     * A return value of {@code -1} means that the information is not
```

Comment:
instead of -1, maybe seeing a constant preferable

Reply:
Added.